.github/workflows/util/execute_and_retry.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+execute_and_retry () {
```

Comment:
Put a comment to say what this is used for and how to use it. What happens if we use this function to run something like `terraform apply`? If it fails it will retry without running `terraform destroy` first, right?

Reply:
There is a third parameter to perform cleanup action. Will mention in a comment

Reply:
Ah makes sense, thanks